evm/vm/forks/sharding/vmc_handler.py

```diff
@@ -1,3 +1,6 @@
+from typing import (  # noqa: F401
+    Dict
+)
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically